invenio_app_rdm/theme/views.py

```diff
@@ -63,6 +64,11 @@ def index():
     )
 
 
+def robots():
```

Comment:
What is the advantage of this approach vs having a static file in the cookiecutter repo? For simple configurations a config variable is okay. However, could this config become large enough to be moved to a separate file? if so, it would not be supported?

For the Describo profile, we also need to serve a static file (which is similar to this - same outcome, different content -). IIRC, it was placed in `static` and served via Nginx. This way, we also **offload the serving of the file** to Nginx and avoid passing these requests through the application. CC @alejandromumo for more details.

Reply:
same suggestion from me, https://github.com/inveniosoftware/invenio-app-rdm/pull/1938#discussion_r995498843 I think we should go with simple static file

Reply:
The reason for having this proposal via config (vs. static file via `nginx`) is mainly deployment complexity. We don't have any similar file in cookiecutter (besides static assets) that get served via `nginx`. This means we have to modify cookiecutter + `invenio-cli` + helm charts (?) for getting this in.

Reply:
I do agree that maybe there are some more implications in terms of deployment. But changes should not be too big, we already have a mount for the [invenio assets](https://github.com/inveniosoftware/helm-invenio/blob/30ec941b6b8ebd60472787cd7f08495e9c0bc7de/invenio/templates/deployments/web.yaml#L118) that need to be served. I guess we can profit from this (or follow the same approach). In any case, we will need to do this work to serve the _describo_ profile?

To make a more informed decission. Today, how many requests are done to the robots.txt in Zenodo or CDS? if there ar every few I guess passing it by the application would be okay.

Reply:
From Zenodo ~2 weeks of access logs we have ~3.5k requests to `/robots.txt` per day (2.5 req/min). IMHO that's low in terms of load, so adding the complexity of serving via nginx seemed overkill. I understand the thought process behind it, since `robots.txt` is a bit special and lies somewhere between deployment and business logic (i.e. "what pages/content of my instance do I want Google to index?").

For Describo, we'll rely on the existing mechanism of the cookiecutter `static` folder symlinking, i.e. there's not new top-level endpoint added, just a new `/static/describo` folder.